docs/how-to/debug-with-ssh.md

```diff
@@ -0,0 +1,56 @@
+# How to debug with ssh
+
+The charm exposes an integration `debug-ssh` interface which can be used with
+[tmate-ssh-server charm](https://charmhub.io/tmate-ssh-server/) to pre-configure runners with
+environment variables to be picked up by [action-tmate](https://github.com/canonical/action-tmate/)
+for automatic configuration.
+
+## Prerequisites
+
+To enhance the security of self-hosted runners and its infrastracture, only authorized connections
+can be established. Hence, action-tmate users must have
+[ssh-key registered](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
+on the GitHub account.
+
+For users using Canonical's self-hosted runners, access to tmate-ssh-server is provided only within
+Canonical VPN. 
```

Comment:
That is probably an internal detail that shouldn't be in the docs?

Reply:
Removed, thank you!